CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - New `edge`-tagged images are published to DockerHub on every master branch
   build.
   [cyberark/conjur#1617](https://github.com/cyberark/conjur/issues/1617)
+- New unit Tests and Integration Tests for audit, and corrects a couple of issues found with their addition.
```

Comment:
can we change this to `Correct a couple of issues found with audit tests`?
and move it to `Changed`

Reply:
done

Reply:
thanks